### ISSUE TITLE
SubmanifoldConvolution get nan

### ISSUE THREAD

**Issue by xuyongzhi** [facebookresearch/SparseConvNet#105]
I got nan when bs=1, feeding a special data. It is fine when I feed most other datas, or delete all batch norm layers. Have no idea why this data cause batch norm nan. When I delete bn layers, the net can get good performance on this data.

**Comment by btgraham** [comment]
Hello. If you are training with batchnorm and the batchsize ==1, then batchnorm may be undefined due to division by zero trying to calculate the sample standard deviation.

**Comment by xuyongzhi** [comment]
Thanks for replying. It is because of batchnormalizaiton. I add track_running_stats = True function, disable it solves the problem.
